### PR TITLE
MANUAL

### PR DESCRIPTION
**Pull request by michihdeu**

### CODE REVIEW

**Comment by michihdeu** [comment]
@jteresco could you bring that live soon so that I can check the links - especially participate.php. Thanks!

File: participate.php [hunk 1]
```diff
@@ -89,6 +89,9 @@
         to open the info window and copy the label name <code>MO/IL</code>.</li>
         <li>The waypoints at the ends of the traveled segment may be listed in either order.</li>
       </ul></br>
+      <font color="#ff0000"><b>NEW</b></font> The <code>.list Tool</code> option helps generating the full <code>.list</code> file line
+      by clicking on the first and last segment of your travel.</br>
```
Comment (yakra) [inline participate.php:93]: clicking on the first and last segment creates .list entries for these individual segments, not a unified .list entry including both of these as well as intermediate segments.
A user would instead click on the 1st segment and then shift+click on the 2nd.

Comment (yakra) [reply participate.php:93]: Everything other than this detail looks good to me.

Comment (michihdeu) [reply participate.php:93]: @yakra I don't see what you suggest (which line etc.) but it sounds good. Go ahead and make the change!

**Comment by jteresco** [comment]
It's up.

File: participate.php (hunk 1 above)
Comment (michihdeu) [reply participate.php:93]: @yakra Did you mean I should change
`The .list Tool option helps generating the full .list file line by clicking on the first and last segment of your travel.`
to
`The .list Tool option helps generating the full .list file line by clicking on the first and last segment creates .list entries for these individual segments, not a unified .list entry including both of these as well as intermediate segments.
A user would instead click on the 1st segment and then shift+click on the 2nd.`

I think that the first part makes sense but `not a unified .list entry including both of these as well as intermediate segments.
A user would instead click on the 1st segment and then shift+click on the 2nd.`
Not sure what you want to say with the "unified entry" and "intermediate segments".

But if you say it's fine, I'll change it.

@jteresco what do you think?

Comment (michihdeu) [reply participate.php:93]: ok. leave as-is. Open a new issue if required

**Pull request merged**